packages/account/src/Components/trading-assessment/trading-assessment-form.jsx

```diff
@@ -0,0 +1,197 @@
+    };
+
+    const isAssessmentCompleted = answers => Object.values(answers).every(answer => Boolean(answer));
+
+    return (
+        <div className={classNames('trading-assessment', class_name)}>
+            <Text as='p' color='prominent' size='xxs' className='trading-assessment__side-note'>
+                <Localize i18n_default_text='In providing our services to you, we are required to obtain information from you in order to assess whether a given product or service is appropriate for you.' />
+            </Text>
+            <section className='trading-assessment__header'>
+                <div className='trading-assessment__header--background'>
+                    <Button
+                        onClick={displayPreviousPage}
+                        transparent
+                        style={hideElement(
+                            !is_header_navigation || current_question_details.current_question_index === 0
+                        )}
+                        type='button'
+                    >
+                        <Icon icon='IcChevronLeft' color='black' />
+                    </Button>
+                    <Text as='h1' color='prominent' weight='bold' size='xs'>
+                        {current_question_details.current_question_index + 1} {localize('of')}{' '}
+                        {assessment_questions.length}
+                    </Text>
+                    <Button
+                        onClick={displayNextPage}
+                        transparent
+                        is_disabled={!is_next_button_enabled}
+                        className={classNames({ 'disable-pointer': !is_next_button_enabled })}
+                        style={hideElement(
+                            !is_header_navigation ||
+                                current_question_details.current_question_index === last_question_index
+                        )}
+                        type='button'
+                    >
+                        <Icon icon='IcChevronRight' color={is_next_button_enabled ? 'black' : 'secondary'} />
+                    </Button>
+                </div>
+                <div className='trading-assessment__header--arrow-down' />
+            </section>
+            <section className='trading-assessment__form'>
+                <Formik initialValues={{ ...form_value }} onSubmit={(values, action) => onSubmit(values, action)}>
+                    {({ setFieldValue, values }) => {
+                        const { question_text, form_control, answer_options, questions } =
+                            current_question_details.current_question;
+
+                        return (
+                            <Form className='trading-assessment__form--layout'>
+                                <div className='trading-assessment__form--fields'>
+                                    {questions && questions.length ? (
+                                        <TradingAssessmentDropdown
```

Comment:
can we select dropdown and radio button based on the field_type property in trading-assessment-config?